modules/devkit-support/src/main/java/org/mule/security/oauth/processor/BaseOAuth2AuthorizeMessageProcessor.java

```diff
@@ -134,12 +136,16 @@ private void setState(Map<String, String> extraParameters, MuleEvent event)
 
                 try
                 {
+                    JsonProperty property=field.getAnnotation(JsonProperty.class);
```

Comment:
I don't think this code belongs here. devkit-support module is annotations agnostic (actually, devkit annotations have compile retention and mule does not and will not depend on them).

The AuthorizationParameter are obtained by calling moduleObject.getDefaultUnauthorizedConnector().getAuthorizationParameters();

So, if what you want to specify aliases for them, then you should add code on the connector itself so that parameter.getName() returns the alias instead of the real name. This provides better compatibility between different connector and mule versions.

Notice that if you build a connector in which you use the alias functionality and deploy it on a mule version which doesn't have this improvement, then your alias would not work. If you do it on the connector, you'll have better portability.

In summary, I think that this is a devkit concern, not a mule one. If you agree, I'll close both this PR and the Jira issue